src/Yasumi/Provider/Brazil.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ *  This file is part of the Yasumi package.
+ *
+ *  Copyright (c) 2015 - 2016 AzuyaLabs
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ *  @author Dorian Neto <doriansampaioneto@gmail.com>
+ */
+namespace Yasumi\Provider;
+
+use DateInterval;
+use DateTime;
+use DateTimeZone;
+use Yasumi\Holiday;
+
+/**
+ * Provider for all holidays in the Brazil.
+ */
+class Brazil extends AbstractProvider
+{
+
+    use CommonHolidays, ChristianHolidays;
+
+    /**
+     * Initialize holidays for the Brazil.
+     */
+    public function initialize()
+    {
+        $this->timezone = 'America/Fortaleza';
+
+        // Add common holidays
+        $this->addHoliday($this->newYearsDay($this->year, $this->timezone, $this->locale));
+        $this->addHoliday($this->internationalWorkersDay($this->year, $this->timezone, $this->locale));
+
+        // Add Christian holidays
+        $this->addHoliday($this->christmasDay($this->year, $this->timezone, $this->locale));
+        $this->addHoliday($this->easter($this->year, $this->timezone, $this->locale));
+        $this->addHoliday($this->corpusChristi($this->year, $this->timezone, $this->locale));
+        $this->addHoliday($this->goodFriday($this->year, $this->timezone, $this->locale));
+
+        /**
+         * Carnaval
```

Comment:
Would you mind providing English description for these holidays? :) Most of the users are English speaking and would not necessarily understand Brazilian Portuguese.